### DISCUSSION TITLE
Suggestion: be more friendly for Qubes template-based VMs

### DISCUSSION THREAD

**Discussion by arkenoi** [pry0cc/axiom#314] [Ideas]
Hi,

my typical pentest engagement VM is based on kali-rolling template running on Qubes OS. It is a bit tricky to run Axiom here, because all changes to the root filesystem are not persistent and do not survive reboot, to make something permanent I need to put it to /usr/local or /home. May I suggest moving all non-system dependencies (everything not installed via the package manager) to /usr/local to keep the installation more clean?

**Comment by 0xtavian** [comment]
@arkenoi thanks for opening a discussion! I like that idea but I am not sure if I will have the time in the next few months to get around to it. We just merged a new feature in axiom-build that allows users to supply a custom Packer JSON provisioner. I would recommend copying Default https://github.com/pry0cc/axiom/blob/master/images/provisioners/full.json and try to alter the appropriate commands to suit your needs. After you are done feel free to submit a pull request with the updated Packer JSON under a different name.

**Reply by arkenoi** [reply]
Ah, i was referring not to the scanning instances in the cloud, but rather to the Axiom control environment on the workstation. Yet I did those modifications for myself, maybe I will tidy it a little bit and make a pull request.
